### ISSUE TITLE
unable to render `02_dehyphenate.qmd`

### ISSUE THREAD

**Issue by nicholasjhorton** [STAT325-S24/HistoryAmherstCollege#49]
When I try to render `02_dehyphenate.qmd` I got the following error message:

```
processing file: 02_dehyphenate.qmd
  |.............................................       |  86% [unnamed-chunk-3]
Quitting from lines 38-53 [unnamed-chunk-3] (02_dehyphenate.qmd)
Error in `if (str_detect(chapter_lines[i], "-$")) ...`:
! missing value where TRUE/FALSE needed
Backtrace:
 1. global fix_up_lines(chapter_name)
                                                                                                            
Execution halted
```

@arogers24 and @FranciscoJFM02 can you please confirm that this is working on your end? Any guidance that you might be able to offer would be greatly appreciated as I'm stuck at present.

**Comment by nicholasjhorton** [comment]
It would be really helpful if the function indicated where it was when it ran into issues and problems. Is that feasible?

**Comment by nicholasjhorton** [comment]
@arogers24 and @FranciscoJFM02 can you please investigate and comment prior to Tuesday's standup? Thanks in advance, Nick

**Comment by nicholasjhorton** [comment]
Here's what I see what I added some debugging assistance:

```
Quitting from lines 41-56 [unnamed-chunk-3] (02_dehyphenate.qmd)
Error in `fix_up_lines()`:
! Error: find_match is NA at line 551of file /Users/nicholashorton/Dropbox/STAT325S24/git/HistoryAmherstCollege/data-raw/../data-raw-depaginate/chapter16_cleaned.txt
Backtrace:
 1. global fix_up_lines(chapter_name)
 ```

**Comment by nicholasjhorton** [comment]
The issue here seemed to be in the depaginator, which left the following in https://github.com/STAT325-S24/HistoryAmherstCollege/blob/main/data-raw-depaginate/chapter16_cleaned.txt

```
"But the very first week of labor," we quote from his journal,
"demonstrated the necessity of immediate suspension. I yes- 
 
 
 
294 HISTORY OF AMHERST COLLEGE.

terday (September 26) held my last exercise with ray class.  
```

My only speculation is that the double quotes are somehow messing with us in some unusual way.

**Comment by nicholasjhorton** [comment]
Making progress on this front.

See https://github.com/STAT325-S24/HistoryAmherstCollege/commit/cbdb6ebec3567a54f45bee44446f07d0040939cc

**Comment by nicholasjhorton** [comment]
More edits: https://github.com/STAT325-S24/HistoryAmherstCollege/commit/3d6f46b709b1b72d1c6b5ccc814be9f02d0e28c6

**Comment by nicholasjhorton** [comment]
Closed in favor of #52
